src/order/lattice.lean

```diff
@@ -395,6 +402,13 @@ lemma inf_inf_distrib_left (a b c : α) : a ⊓ (b ⊓ c) = (a ⊓ b) ⊓ (a ⊓
 lemma inf_inf_distrib_right (a b c : α) : (a ⊓ b) ⊓ c = (a ⊓ c) ⊓ (b ⊓ c) :=
 @sup_sup_distrib_right αᵒᵈ _ _ _ _
 
+lemma inf_eq_inf_of_le_of_le (h1 : a ⊓ c ≤ b) (h2 : b ⊓ c ≤ a) : a ⊓ c = b ⊓ c :=
+(le_inf h1 inf_le_right).antisymm (le_inf h2 inf_le_right)
+
+lemma inf_eq_inf_iff_le_le : a ⊓ c = b ⊓ c ↔ a ⊓ c ≤ b ∧ b ⊓ c ≤ a :=
+⟨λ h, ⟨by {rw h, exact inf_le_left},by {rw ←h, exact inf_le_left}⟩,
```

Comment:
Use the `order_dual` trick?

Reply:
Thanks for the suggestions! All changed.